.circleci/config.yml

```diff
@@ -275,6 +303,73 @@ jobs:
       - store_test_results:
           path: libs/SalesforceHybrid/build/outputs/androidTest-results/
 
+  generate-artifacts:
+    <<: *defaults
+    steps:
+      - checkout
+      - restore_cache: *restore-gradle-cache
+      - restore_cache: *restore-node-cache
+      - restore_cache: *restore-ruby-cache
+      - run: *setup-env
+      - save_cache: *save-node-cache
+      - save_cache: *save-ruby-cache
+      - run: *compile-libraries
+      - store_artifacts:
+          path: libs/SalesforceAnalytics/build/outputs/aar/
+          destination: Libraries
```

Comment:
Could we use lowercase `libraries` as the destination since the others are lowercase too, i.e. `native-apps` and `hybrid-apps`?